scripts/machine-setup.ps1

```diff
@@ -3,10 +3,11 @@
 
 $VcpkgInstallDirectory = Join-Path -Path $HOME -ChildPath "vcpkg"
 $DotNetInstallDirectory = Join-Path -Path $HOME -ChildPath "dotnet"
+$ProgressPreference = 'SilentlyContinue'
 
 if (!(Get-Command python -ErrorAction SilentlyContinue ))
 {
-  $PythonInstaller = New-TemporaryFile | Rename-Item -NewName { $_.Name + ".exe" } - PassThru
```

Comment:
What actually changed here?

Reply:
There was a typo that could potentially cause issues in Powershell - not world ending as the command would run, but removing the space between `-` and `PassThru` alleviates the potential issue.